app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/actions.ts

```diff
@@ -240,6 +261,35 @@ export const getSubmissionsByFormat = async ({
             } as Answer;
           }
 
+          // Handle "Split" AddressComplete in a similiar manner to dynamic fields.
+          if (
+            question?.type === FormElementTypes.addressComplete &&
+            question.properties.addressComponents?.splitAddress === true
+          ) {
+            const addressObject = JSON.parse(answer as string) as AddressElements;
+
+            const questionComponents = question.properties.addressComponents as AddressComponents;
+            if (questionComponents.canadianOnly) {
+              addressObject.country = "CAN";
+            }
+
+            const reviewElements = getAddressAsAnswerElements(
+              question,
+              addressObject,
+              extraTranslations
+            );
+            const addressElements = [reviewElements];
+
+            return {
+              questionId: question.id,
+              type: FormElementTypes.address,
+              questionEn: question?.properties.titleEn,
+              questionFr: question?.properties.titleFr,
+              answer: addressElements,
+            } as Answer;
+          }
+
+          // return the final answer object
```

Comment:
I don't think you should be doing this formatting here, it should be handled in the getAnswerAsString function.

And this is currently broken- when I try to download an html formatted Response, I get the following error from the action: `ERROR: Could not create submissions in format html: Cannot read properties of undefined (reading 'en')`

Reply:
I suppose it's because you want to sometimes return an object instead of a string? Either way I believe it'll be a string- as JSON will need to be serialized anyway.

Reply:
It's to return an object allowing a piggy backing off the same functionality as the dynamic rows.